.azuredevops/policies/approvercountpolicy.yml

```diff
@@ -0,0 +1,19 @@
+name: approver_count

```

Comment:
I don't think we need this file in public repo

Reply:
correct, please remove

Reply:
Done! I removed all other onebranch template file changes, unnoticed this one, thanks for noticing it.